.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -15,7 +15,7 @@
 -->
 # {runtime-version} / {sdk-version}
 
-1.  - [ ] Create a page at ".NET Core A&D" OneNote -> "Servicing" section -> "{Month} {YYYY}" to take ongoing notes on problems with the process, workarounds, and fixes.
+1.  - [ ] Create a page at ".NET Core A&D" OneNote -> "Servicing" section -> "Post Mortem" -> "{Month} {YYYY}" to take ongoing notes on problems with the process, workarounds, and fixes.
```

Comment:
```suggestion
1.  - [ ] Create a page at ".NET Core A&D" OneNote -> "Servicing" section -> "Retrospective" -> "{Month} {YYYY}" to take ongoing notes on problems with the process, workarounds, and fixes.
```

Post mortem isn't wrong but might have some implications. (In my mind I link it to "something went wrong" but I don't think that has to be the case.) Let's rename the section to the term we've actually been using.